### ISSUE TITLE
After update of tC, all progress indication is lost

### ISSUE THREAD

**Issue by benjore** [unfoldingWord/translationCore#1333]
The highlighted text is still all there, but there are no green checkmarks and no progress in the progress circles. (Updating from 0.6.13 -> 0.6.15)
Only for tN tool. tW works fine.

**Comment by benjore** [comment]
https://git.door43.org/Danjuma_Alfred_H/sw_tit_text_ulb

**Comment by cckozie** [comment]
The progress, check marks, edits, and comments are all now persisted through updates in both tN and tW tools.

**Comment by cckozie** [comment]
This is still a problem in 0.6.19. The highlighting is maintained through the upgrade, but the check marks and progress are not. (I did the checking in 0.6.18, then upgraded to 0.6.19)

**Comment by mannycolon** [comment]
@cckozie you need to delete your apps folder inside your project folder. going forward this is not going to be an issue with the code I added therefore no deleting of this folder will be needed.

**Comment by cckozie** [comment]
My sequence of events:
Uninstalled tC 
Deleted the translationStudio folder 
Installed 0.6.20
Made some checks in both tN and tW
Installed 0.6.21
All selects, edits, and comments appear to persevere through the upgrade, but none of the visual indicators in the checks menu are present after the upgrade.

**Comment by cckozie** [comment]
Accidentally closed this before.

**Comment by mannycolon** [comment]
@cckozie you have to delete the `apps` folder from the project folder that you're using to test the app.

**Comment by cckozie** [comment]
@mannycolon I deleted the whole translationStudio folder before starting the test.

**Comment by mannycolon** [comment]
@cckozie where is the project youre using to test??how are you loading it??
and can i see the project data?

**Comment by cckozie** [comment]
@mannycolon In my initial test this does appear to work now when upgrading from 0.6.21 to 0.6.22
I need to do more testing on it, but don't work on it yet.

**Comment by cckozie** [comment]
In upgrade from 0.6.23 to 0.6.24 I lost the progress and check indicators in tW (the checks were in the God group). Those in tN did persist through the upgrade. After closing and reopening tC, they did appear again.
[es-419_eph_text_ulb.zip](https://github.com/unfoldingWord-dev/translationCore/files/996880/es-419_eph_text_ulb.zip)
It looks to me like this really is not a problem with the upgrade, but rather making sure the icons are all updated when the page is opened. I'm going to leave this on hold until #1508 is addressed.

**Comment by cckozie** [comment]
I have not seen this problem in 0.6.26.5
